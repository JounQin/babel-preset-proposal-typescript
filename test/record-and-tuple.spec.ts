import _ from './record-and-tuple'
import { execute } from './helpers'

const proposal = 'record-and-tuple'

test(proposal, () => {
  expect(() => execute(proposal)).toThrowErrorMatchingSnapshot()
  expect(_).toEqual(['Record & Tuple proposal', 'tc39'])
})
