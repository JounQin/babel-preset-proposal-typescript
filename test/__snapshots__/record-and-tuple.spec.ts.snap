// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`record-and-tuple 1`] = `"Invalid or unexpected token"`;
